velox/exec/tests/StreamingAggregationTest.cpp

```diff
@@ -345,6 +345,84 @@ class StreamingAggregationTest : public OperatorTestBase {
           .assertResults(sql);
     }
   }
+
+  void assertResult(
+      const RowVectorPtr& expected,
+      const core::PlanNodePtr& plan,
+      uint32_t outputBatchSize) {
+    auto actual = AssertQueryBuilder(plan)
+                      .config(
+                          core::QueryConfig::kPreferredOutputBatchRows,
+                          std::to_string(outputBatchSize))
```

Comment:
Is std::to_string needed?

Reply:
Sorry to make the *copy without understanding* mistake that I forgot to use the new config API I added recently. I Will fix it.